### PR TITLE
Fix for setNetworkConnection API

### PR DESCRIPTION
**Pull request by SrinivasanTarget**
Fixes: https://github.com/appium/appium/issues/5256

### CODE REVIEW

**Comment by SrinivasanTarget** [comment]
@imurchie @jlipps @TikhomirovSergey Please review. This fixes the API to work on all the Android API till API Level 23 (Tested from 19 to 23)

File: lib/commands/touch.js [hunk 1]
```diff
@@ -24,10 +24,10 @@ commands.doTouchAction = async function (action, opts) {
         opts.duration = 1000;
       }
       return await this.touchLongClick(opts.element, opts.x, opts.y, opts.duration);
-    case 'cancel':
-      // TODO: clarify behavior of 'cancel' action and fix this
-      log.warn("Cancel action currently has no effect");
-      break;
+    case 'cancel':
```
Comment (imurchie) [inline lib/commands/touch.js]: This file doesn't need to be in here (it's just whitespace that's changed).

File: lib/commands/network.js [hunk 2]
```diff
@@ -38,7 +38,8 @@ commands.setNetworkConnection = async function (type) {
   });
   if (!airplaneMode) {
     await this.wrapBootstrapDisconnect(async () => {
-      await this.adb.setWifiAndData({wifi, data});
+      await this.adb.setWifiState({wifi});
+      await this.adb.setDataState({data});
```
Comment (imurchie) [inline lib/commands/network.js]: 👍

**Comment by SrinivasanTarget** [comment]
Raised here: https://github.com/appium/appium-android-driver/pull/148

**Pull request closed without merging**